app/models/event.rb

```diff
@@ -362,6 +362,16 @@ def update_rsvp_counts
     )
   end
 
+  def current_state
```

Comment:
this replicates methods provided by `enum current_state` and is blowing everything up. remove it?